src/models/chat/index.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * The action types accepted by the Analytics Events API.
+ *
+ * @public
+ */
+export const ACTIONS = [
+  'ADD_TO_CART',
```

Comment:
can you make these:
```
ADD_TO_CART: 'ADD_TO_CART',
```

So they can be references like `Actions.ADD_TO_CART`?

Reply:
I can do that. Just out of curiosity, what's the benefit of doing it that way?

Reply:
I made this change and now you have to import the `Actions` enum in order to use `.report()`, e.g. `analytics.report({ action: Actions.ADD_TO_CARD })`

To me this feels a lot more annoying than just using the string literal type, for which I get nice intellisense. Is there a benefit to doing this with enums?

Reply:
So the const array isn't a string literally type. I think you could also do `type Actions 'FOO|BAR` which might be more what you want?  I do the enum style in javascript mostly, sometimes I forget about the niceties in typescript.